Lib/sysconfig.py

```diff
@@ -731,7 +756,7 @@ def _main():
         return
     print('Platform: "%s"' % get_platform())
     print('Python version: "%s"' % get_python_version())
-    print('Current installation scheme: "%s"' % _get_default_scheme())
+    print('Current installation scheme: "%s"' % get_default_scheme())
```

Comment:
Would be a good opportunity to migrate this to f-strings?

Reply:
Probably, I’ll combine this to other needed changes, if there are any.